### DISCUSSION TITLE
V5 is coming!

### DISCUSSION THREAD

**Discussion by cyp3rius** [VirtusLab/strapi-plugin-navigation#422] [Ideas]
Hello Folks!

As Strapi V5 beta was published we're already started the v5 re-implementation of the plugin :) 

It's time to ask you, the community, what features you're missing the most and would like to see in upcoming releases of the plugin? :)

We're going to build a community driven backlog combined with our own we're working on right now. Time start!

**Comment by krystianadchitects** [comment]
@cyp3rius Hi there.

Is there any timeline for the V5 plugin release?

Thanks for Your work guys. 🙇

**Reply by cyp3rius** [reply]
I would say late August :)

**Comment by colonder** [comment]
Definitely import/export fix

**Comment by cyp3rius** [comment]
The `3.0.0-beta.0` is live  🚀 And covers all features from `2.x+` but we've changed the way to link `NavigationItem` with `ContentType` so it now should be easier to export / import (polymorphic relations).  

Keep in mind that such change is a **⚠️ breaking** one.

**Reply by TimFoerster** [reply]
Keep up with the good work

**Reply by arku3** [reply]
Is "WRAPPER" item deprecated in 3.0.0?

**Reply by cyp3rius** [reply]
Was reverted back in `3.0.0-beta.4` :)
